core/src/test/java/com/sinch/sdk/core/http/HttpMapperTest.java

```diff
@@ -97,19 +98,20 @@ void serializeDefaultJSON() throws ApiException {
   }
 
   @Test
-  void serializeUninitializedRequired() throws ApiException {
+  void serializeUninitializedRequiredWithDefined() throws ApiException {
```

Comment:
It took me a moment to understand that `WithDefined` refers to the deprecated way to handle the not-initialized variables. Maybe you can rename it with `WithDefinedFlag`

Reply:
Good catch 
Done, but my preference was to refactor test file with:
- xxxWithDefinedGetter
- xxxWithOptionalValueGetter
Seems to better fit with the purpose of tests coverage